### ISSUE TITLE
No Tap-to-Run or Snapshots appear in Govee2MQTT

### ISSUE THREAD

**Issue by eboline** [wez/govee2mqtt#282]
### Govee Device SKU

Multiple Govee Devices

### Govee2MQTT Version

2024.07.13-82ddc6e9

### Describe the issue

At one time I had 3 Tap-to-Run scenes showing in my Govee2MQTT integration.  I have since added a few and none of them are showing up

### Startup Diagnostics

N/A

### Anything else?

I have tried removing the add-on and re-installing, purging the cache, etc. and none of the tap-to-run scenes I have set up will show up.  I can see all of the Govee devices I have set up in the app in Govee2MQTT, but none of the scenes are showing up anywhere in HA.

**Comment by eboline** [comment]
Update:  I had one tap-to-run scene that was a "Drewmview" scene including the H706A lights and the H7053 (not yet supported).  When I deleted that, they all showed up and worked.  What's really interesting is that a different scene I have does include both of those products did work even though the H7053 isn't yet supported, but it wasn't a "DreamView" scene so that appears to be the difference.

Maybe this issue should be converted to a request to add support for the H7053 (Outdoor Ground Lights 2) product.

**Comment by CamSoper** [comment]
@eboline Hmmm... I'm having a similar experience. Used to see my tap-to-runs, now I don't. Deleted all my scenes, including Dreamviews, still no tap-to-runs.

Appropriately, I'm trying to figure this out because I have the same ground lights.

**Comment by CamSoper** [comment]
I also have an update! I rolled back to [2024.04.29-30cf7732](https://github.com/wez/govee2mqtt/releases/tag/2024.04.29-30cf7732) and the ground lights suddenly had an effects dropdown. I rolled forward to  [2024.07.13-82ddc6e9](https://github.com/wez/govee2mqtt/releases/tag/2024.07.13-82ddc6e9) and it continued to work.

🤷🏻‍♂️

**Comment by eboline** [comment]
> I also have an update! I rolled back to [2024.04.29-30cf7732](https://github.com/wez/govee2mqtt/releases/tag/2024.04.29-30cf7732) and the ground lights suddenly had an effects dropdown. I rolled forward to [2024.07.13-82ddc6e9](https://github.com/wez/govee2mqtt/releases/tag/2024.07.13-82ddc6e9) and it continued to work.
> 
> 🤷🏻‍♂️

Does this mean you were able to get the scenes to work again as well?

None of my devices have an effects dropdown list and it sounds like they should based on what you are saying.  At least the ground lights for sure.  How did you "roll back" to the previous version because I might give that a try and see if the effects drop down shows up.

**Comment by CamSoper** [comment]
Yeah, all my devices have effects dropdowns and my Govee tap-to-runs show up correctly now, too.

I use Docker to host govee2mqtt, so I changed my `docker-compose.yml` to point to the tag instead of `latest`.  Then run `docker-compose pull` and `docker-compose up -d` to restart it on the new (older) image. Everything started working, so I rolled forward to `latest` again. Weirdly, everything *kept* working so I haven't touched it since.

The nice thing about running something like this in Docker is you can throw away the entire machine and start from scratch very easily. If you're not already, I highly recommend it.

**Comment by eboline** [comment]
Can you send a screenshot of what you see with your ground lights?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by CamSoper** [comment]
![image](https://github.com/user-attachments/assets/b9b809a6-da16-4402-8a1a-12d0f486b350)

![image](https://github.com/user-attachments/assets/e197853b-6753-48db-83da-a8bc1e58f5f7)

**Comment by eboline** [comment]
Yep, as I suspected, I'm an idiot!

I have that.  I was looking for it on the main screen for the device where all the segments are listed.  Thank you!!!

Do any of your tap-to-runs include "Dreamview" scenes you put together?  Are those showing up correctly now too?

**Comment by CamSoper** [comment]
Yes and yes

**Comment by eboline** [comment]
That's great news!!!
